pages/user.js

```diff
@@ -1,10 +1,22 @@
+/* eslint-disable no-unused-vars */
 import React, { Component } from 'react';
 import { Row, Col } from 'antd';
 import fetch from 'isomorphic-unfetch';
+import io from 'socket.io-client';
 import Layout from '../components/layout';
 import './admin/profile/index.scss';
+import SOCKET_IO_EVENTS from '../server/lib/config/constants';
```

Comment:
refer to the coding convention for how to order these imports

Reply:
Not sure there is something in the coding convention regarding the import

Reply:
You are right, I forgot to add it. Here is it though
```
// external dependencies in alpha-numeric order

// internal dependencies in alpha-numeric order

// constant declarations
```